Exception/CmpException.php

```diff
@@ -29,16 +29,39 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  * @author Nosto Solutions Ltd <contact@nosto.com>
- * @copyright 2020 Nosto Solutions Ltd
+ * @copyright 2021 Nosto Solutions Ltd
  * @license http://opensource.org/licenses/BSD-3-Clause BSD 3-Clause
  *
  */
-
 namespace Nosto\Cmp\Exception;
 
 use Exception;
 
 class CmpException extends Exception
 {
+    /**
+     * CmpException constructor.
+     * @param string $message
+     * @param int|string $storeId
+     * @param string $currentUrl
+     */
+    public function __construct(
+        $message = "",
```

Comment:
Why are there default values here?

Reply:
I added these in case the storeId or currentUrl are not passed from where the Exception is thrown.